src/AvtaleSide/steg/GodkjenningSteg/Oppsummering/OppsummeringMentor/OmMentorOppsummering.tsx

```diff
@@ -0,0 +1,65 @@
+import { Element } from 'nav-frontend-typografi';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import BEMHelper from '@/utils/bem';
+import { Mentorinfo } from '@/types/avtale';
+import SjekkOmVerdiEksisterer from '../SjekkOmVerdiEksisterer/SjekkOmVerdiEksisterer';
+import Stegoppsummering from '../Stegoppsummering/Stegoppsummering';
+import { Column, Container, Row } from 'nav-frontend-grid';
+import VerticalSpacer from '@/komponenter/layout/VerticalSpacer';
+import { Avtalepart } from '@/AvtaleSide/steg/GodkjenningSteg/Oppsummering/Avtaleparter/Avtaleparter';
+import { ReactComponent as MentorIkon } from '@/assets/ikoner/mentor.svg';
+
+const cls = BEMHelper('mentorOppsummering');
+
+const verdi = (tall: number) => {
+    return tall === null ? '' : tall.toString();
+};
+
+const OmMentorOppsummering: FunctionComponent<Mentorinfo> = ({
+    mentorFornavn,
+    mentorEtternavn,
+    mentorOppgaver,
+    mentorTimelonn,
+    mentorAntallTimer,
+}) => {
```

Comment:
Jeg synes komponenter blir finere og mer oversiktlig når man ikke destructer propsene, men heller skriver slik:

`const OmMentorOppsummering: FunctionComponent<Mentorinfo> = props => {
`

og heler da skriver `props.mentorFornavn` osv. Blir også lettere å se hva som kommer fra `props` og ikke. Forøvrig ikke noen stor sak.
Enig, @mbolstad ?